fineract-core/src/main/java/org/apache/fineract/batch/service/BatchApiServiceImpl.java

```diff
@@ -156,11 +159,16 @@ private void callRequestRecursive(BatchRequest request, BatchRequestNode request
                 BatchRequest resolvedChildRequest;
                 try {
                     resolvedChildRequest = this.resolutionHelper.resoluteRequest(childNode.getRequest(), response);
+                } catch (JsonPathException jsonPathException) {
```

Comment:
Please instead of this use error mapping instead. See `InvalidJsonExceptionMapper`